docs/tutorials/flow-web3-app.mdx

```diff
@@ -0,0 +1,453 @@
+      cadence: `
+        import Profile from 0xProfile
+
+        transaction {
+          prepare(account: AuthAccount) {
+            // Only initialize the account if it hasn't already been initialized
+            if (!Profile.check(account.address)) {
+              // This creates and stores the profile in the user's account
+              account.save(<- Profile.new(), to: Profile.privatePath)
+
+              // This creates the public capability that lets applications read the profile's info
+              account.link<&Profile.Base{Profile.Public}>(Profile.publicPath, target: Profile.privatePath)
+            }
+          }
+        }
+      `,
+      payer: fcl.authz,
+      proposer: fcl.authz,
+      authorizations: [fcl.authz],
+      limit: 50
+    })
+
+    const transaction = await fcl.tx(transactionId).onceSealed()
+    console.log(transaction)
+  }
+
+  // NEW
+  const executeTransaction = async () => {
+    const transactionId = await fcl.mutate({
+      cadence: `
+        import Profile from 0xProfile
+
+        transaction(name: String) {
+          prepare(account: AuthAccount) {
+            account
+              .borrow<&Profile.Base{Profile.Owner}>(from: Profile.privatePath)!
+              .setName(name)
+          }
+        }
+      `,
+      args: (arg, t) => [arg("Flow Developer!", t.String)],
+      payer: fcl.authz,
+      proposer: fcl.authz,
+      authorizations: [fcl.authz],
+      limit: 50
+    })
+
+    fcl.tx(transactionId).subscribe(res => setTransactionStatus(res.status))
+  }
+
+  const AuthedState = () => {
+    return (
+      <div>
+        <div>Address: {user?.addr ?? "No Address"}</div>
+        <div>Profile Name: {name ?? "--"}</div>
+        <div>Transaction Status: {transactionStatus ?? "--"}</div> {/* NEW */}
+        <button onClick={sendQuery}>Send Query</button>
+        <button onClick={initAccount}>Init Account</button>
+        <button onClick={executeTransaction}>Execute Transaction</button> {/* NEW */}
+        <button onClick={fcl.unauthenticate}>Log Out</button>
+      </div>
+    )
+  }
+
+  const UnauthenticatedState = () => {
+    return (
+      <div>
+        <button onClick={fcl.logIn}>Log In</button>
+        <button onClick={fcl.signUp}>Sign Up</button>
+      </div>
+    )
+  }
+
+  return (
+    <div>
+      <Head>
+        <title>FCL Quickstart with NextJS</title>
+        <meta name="description" content="My first web3 app on Flow!" />
+        <link rel="icon" href="/favicon.png" />
+      </Head>
+      <h1>Flow App</h1>
+      {user.loggedIn
+        ? <AuthedState />
+        : <UnauthenticatedState />
+      }
+    </div>
+  )
+}
+
+```
+![Mutation image](../images/4.png)
+
+Now, if you click the `Execute Transaction` button, you will see the statuses update next to `Transaction Status`.
+When you see **4** that means it's sealed! Status code meanings can be found here. If you query the account profile again, `Profile Name:` should now display `Flow Developer`.
```

Comment:
link is missing on "here"-> https://docs.onflow.org/access-api/#transaction-status